network/internal/errors.go

```diff
@@ -1,4 +1,4 @@
-package errors
+package internal
```

Comment:
Goland is telling me this error isn't used anywhere, maybe we can just remove? `InvalidEngineError`

Reply:
Fixed in https://github.com/onflow/flow-go/pull/183/commits/e69cb2fd1ba0edd9d9990f6527de69b893593936. Removed `InvalidEngineError` and made other error inline in code as it was only utilized on a test file.